src/archanaNJun20/BookMeetingRoom.java

```diff
@@ -0,0 +1,105 @@
+package archanaNJun20;
+
+public class BookMeetingRoom {
+	
+	double nalandaHours,takshashilaHours,agraFortHours,pratapGadhHours;
```

Comment:
No need to define hours variables of double type as you have used "int hours" in whole program while passing it as a parameter